### ISSUE TITLE
Can't install elm-analyse on Linux

### ISSUE THREAD

**Issue by j-panasiuk** [stil4m/elm-analyse#57]
I wanted to try out elm-analyse, but unfortunately I have trouble with installation. It seems that the script can't find elm-format-download (whatever that thing is...)
I'm on Linux Mint 18

node 7.2.1
elm 18.0
elm-format 0.5.2-alpha

This is the error I get trying to run `yarn add`
```
error /home/jacek/node_modules/elm-analyse: Command failed.
Exit code: 127
Command: sh
Arguments: -c ./node_modules/.bin/elm-format-download -t ./node_modules/.bin
Directory: /home/jacek/node_modules/elm-analyse
Output:
sh: 1: ./node_modules/.bin/elm-format-download: not found
```

and here is error part from npm-debug.log when I run `(sudo) npm add -g elm-analyse`
```
38875 verbose stack Error: elm-analyse@0.5.0 postinstall: `elm-format-download -t ./node_modules/.bin`
38875 verbose stack spawn ENOENT
38875 verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/lib/utils/spawn.js:33:16)
38875 verbose stack     at emitTwo (events.js:106:13)
38875 verbose stack     at ChildProcess.emit (events.js:191:7)
38875 verbose stack     at maybeClose (internal/child_process.js:885:16)
38875 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
38876 verbose pkgid elm-analyse@0.5.0
38877 verbose cwd /home/jacek/bin
38878 error Linux 4.4.0-21-generic
38879 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "-g" "elm-analyse"
38880 error node v7.2.1
38881 error npm  v3.10.10
38882 error file sh
38883 error code ELIFECYCLE
38884 error errno ENOENT
38885 error syscall spawn
38886 error elm-analyse@0.5.0 postinstall: `elm-format-download -t ./node_modules/.bin`
38886 error spawn ENOENT
38887 error Failed at the elm-analyse@0.5.0 postinstall script 'elm-format-download -t ./node_modules/.bin'.
38887 error Make sure you have the latest version of node.js and npm installed.
38887 error If you do, this is most likely a problem with the elm-analyse package,
38887 error not with npm itself.
38887 error Tell the author that this fails on your system:
38887 error     elm-format-download -t ./node_modules/.bin
38887 error You can get information on how to open an issue for this project with:
38887 error     npm bugs elm-analyse
38887 error Or if that isn't available, you can get their info via:
38887 error     npm owner ls elm-analyse
38887 error There is likely additional logging output above.
```

**Comment by stil4m** [comment]
Should have fixed this in the `0.5.1` release. @j-panasiuk can you verify this?

**Comment by j-panasiuk** [comment]
Yes, it works now.
That was fast, thanks! :)
